starter-web-servlet/build.gradle

```diff
@@ -15,11 +15,11 @@ dependencies {
     implementation project(":starter-api")
     implementation "io.micronaut:micronaut-inject"
 
-    implementation("io.micronaut.servlet:micronaut-servlet-engine:1.0.0.M3")
-    developmentOnly "io.micronaut.servlet:micronaut-http-server-jetty:1.0.0.M3"
+    implementation("io.micronaut.servlet:micronaut-servlet-engine:3.1.0")
```

Comment:
can't we use the version in the bom?

Reply:
`micronaut-servlet-engine` is not in the BOM https://search.maven.org/artifact/io.micronaut/micronaut-bom/3.3.0/pom

It's a transitive dependency of `micronaut-http-server-jetty`, but we need it as an implementation artifact for the war to work 🤔

Reply:
I think what we have to do is update the servlet module build and publish a bom

Reply:
I have created an issue: https://github.com/micronaut-projects/micronaut-servlet/issues/268